frontend/src/component/feature/FeatureStrategy/FeatureStrategyMenu/FeatureStrategyMenu.tsx

```diff
@@ -35,25 +50,51 @@ export const FeatureStrategyMenu = ({
         setAnchor(event.currentTarget);
     };
 
+    const createStrategyPath = formatCreateStrategyPath(
+        projectId,
+        featureId,
+        environmentId,
+        'flexibleRollout',
+        true
+    );
+
     return (
         <div onClick={event => event.stopPropagation()}>
             <PermissionButton
                 permission={CREATE_FEATURE_STRATEGY}
                 projectId={projectId}
                 environmentId={environmentId}
-                onClick={onClick}
+                onClick={() => navigate(createStrategyPath)}
                 aria-labelledby={popoverId}
                 variant={variant}
+                size={size}
                 sx={{ minWidth: matchWidth ? '282px' : 'auto' }}
             >
                 {label}
             </PermissionButton>
+
+            <StyledAdditionalMenuButton
```

Comment:
This should be PermissionIconButton

Reply:
Also should have locking behaviour like this

![image](https://github.com/Unleash/unleash/assets/964450/cc2a54a5-88d0-4a29-97c9-3e752501efdc)

Reply:
Nope. PermissionIconButton doesn't have outline. https://mui.com/material-ui/react-button/#icon-button